subcommands/events/list.go

```diff
@@ -0,0 +1,36 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/cheynewallace/tabby"
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	cmd.AddCommand(&cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List configured event queues",
+		Run:     doList,
+	})
+}
+
+func doList(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+	logrus.Debugf("Listing event queues for: %s", factory)
+
+	queues, err := api.EventQueuesList(factory)
+	subcommands.DieNotNil(err)
+	fmt.Println(queues)
```

Comment:
Do we need to print `queues` twice? One time here and the second time in the loop below.

Reply:
oops - debug slipped in there. i usually do Printl("ANDY", data) so its more obvious :)

Reply:
commit 2a3de1e7598590f507f05c0d729f6d4f2df7826a